frontend/src/metabase/visualizations/visualizations/PivotTable/PivotTableCell.tsx

```diff
@@ -59,10 +62,30 @@ export function Cell({
       }}
       onClick={onClick}
     >
-      <div className={cx("px1 flex align-center", { "justify-end": isBody })}>
-        <Ellipsified>{value}</Ellipsified>
-        {icon && <div className="pl1">{icon}</div>}
-      </div>
+      <>
+        <div className={cx("px1 flex align-center", { "justify-end": isBody })}>
+          <Ellipsified>{value}</Ellipsified>
+          {icon && <div className="pl1">{icon}</div>}
+        </div>
+        {!!onResize && (
```

Comment:
;knit; 
do we need the !! here?

Reply:
Probably not strictly necessary if the function is either defined or undefined (as it should be), but I like this pattern for two reasons

1) I think casting to boolean here make it easy to understand that we're just looking for the truthiness of this variable
2) It makes this check more resilient, in the case of a falsy value that react will render, like `0`

![Screen Shot 2023-01-16 at 12 22 45 PM](https://user-images.githubusercontent.com/30528226/212752945-be4f75e8-4830-4cb8-8987-95285659de50.png) ![Screen Shot 2023-01-16 at 12 23 17 PM](https://user-images.githubusercontent.com/30528226/212752947-7a68ef1c-3466-4911-b4db-900383e67c77.png)

Reply:
Beautiful, onwards then o/